test/Test/Database/LSMTree/Internal/Merge.hs

```diff
@@ -37,15 +36,10 @@ tests = testGroup "Test.Database.LSMTree.Internal.Merge"
           prop_CloseMerge fs level stepSize wbs
     ]
   where
-    withSessionDir = Temp.withSystemTempDirectory "session-merge"
-
     -- TODO: run using mock file system once simulation is merged:
     -- https://github.com/input-output-hk/fs-sim/pull/48
     -- (also check all handles closed, see Test.Database.LSMTree.Internal.Run)
-    ioPropertyWithRealFS prop =
-        ioProperty $ withSessionDir $ \sessionRoot -> do
-          let mountPoint = FS.MountPoint sessionRoot
-          prop (FsIO.ioHasFS mountPoint)
+    ioPropertyWithRealFS prop = ioProperty $ withTempIOHasFS "session-merge" prop
```

Comment:
We should switch this one to the mock `HasFS` soon anyway. So no need to touch it, but I'm okay with leaving the change in, now that it's there.